rotkehlchen/db/utils.py

```diff
@@ -147,33 +147,65 @@ def get(self, blockchain: SupportedBlockchain) -> ListOfBlockchainAddresses:
         raise AssertionError(f'Unsupported blockchain: {blockchain}')
 
 
-class DBAssetBalance(NamedTuple):
+@dataclass(init=True, repr=True, eq=True, order=False, unsafe_hash=False, frozen=False)
+class DBAssetBalance:
     category: BalanceType
     time: Timestamp
     asset: Asset
-    amount: str
-    usd_value: str
+    amount: FVal
+    usd_value: FVal
 
     def serialize(
             self,
             export_data: Optional[Tuple[Asset, Price]] = None,
-    ) -> Dict[str, Union[str, int]]:
+    ) -> Dict[str, Union[str, int, FVal]]:
+        """Serializes a `DBAssetBalance` to dict.
+        If `export_data` is provided, the data is serialized for human consumption i.e
+        `usd_value` is converted to the user's currency & timestamp is converted to a readable one.
+        """
         if export_data:
             return {
                 'timestamp': timestamp_to_date(self.time, '%Y-%m-%d %H:%M:%S'),
                 'category': self.category.serialize(),
                 'asset': str(self.asset),
-                'amount': self.amount,
-                f'{export_data[0].symbol.lower()}_value': str(FVal(self.usd_value) * export_data[1]),  # noqa: 501
+                'amount': str(self.amount),
+                f'{export_data[0].symbol.lower()}_value': str(FVal(self.usd_value) * export_data[1]),  # noqa: E501
```

Comment:
`self.usd_value` is already an fval ...

```suggestion
                f'{export_data[0].symbol.lower()}_value': str(self.usd_value * export_data[1]),  # noqa: E501
```